docs/ruleset_functions/G311_exceptions/zone_meets_G3_1_1c

```diff
@@ -0,0 +1,74 @@
+"average" is not particularly well defined in this exception. 
+for the peak W/sf calculation I've assumed:
+	1. the zone being tested is not included
+ 	2. the average: (each space coincident peak W) / total_area
+
+**Inputs:** 
+- **RMR**
+- **zone**
+
+**Returns:**  
+- **result**: a string - either "C" or "No"
+ 
+**Function Call:**
+- **get_baseline_system_types**
+- **get_zone_peak_internal_load**
+
+## Logic:
+- set the result variable to "No" - only a positive test can give it a different value: `result = "No"`
+- get the building_segment the zone is in:
+`for building_segmentin RMR.building.buildings:`
+	- check if the zone is in the building_segment:
+	`if zone in building_segment.zones:`
+		- break - this means the variable "building_segment" is the building_segment that contains the zone:
+		`break`
+- make a list of the heating_ventilation_air_conditioning_systems in the building_segment:
+`relevant_systems = [a.id for a in building_segment.heating_ventilation_air_conditioning_systems]`
+
+
+
+- first check if the system type is one of the following:
+`eligible_primary_system_types = ["SYS-5","SYS-5b","Sys-6","Sys-6b","Sys-7","Sys-7a","Sys-7b","Sys-7c","Sys-8","Sys-8a","Sys-8b","Sys-8c","Sys-8d","Sys-9","Sys-9b","Sys-10","Sys-11.1","Sys-11.1a","Sys-11b","Sys-11c","Sys-12","Sys-12a","Sys-12b","Sys-12c","Sys-13","Sys-13a"]`
+- get the dict of get_baseline_system_types:
+`system_types_b = get_baseline_system_types(RMR)`
+- create a sub-list of system_types_b that includes only the system types and id references in the building_segment:
+`segment_system_types_b = {key: system_types_b[0] for key in system_types_b if(len(set(system_types_b[key])&set(relevant_systems)) > 0)`
+
+- create set of system types that are eligible for this rule AND included:
+`eligible_systems_in_RMR = (set(segment_system_types_b) & set(eligible_primary_system_types))`
+- check if any of the eligible_primary_system_types is in the system_types_b:
+`if((len(eligible_systems_in_RMR)) > 0):`
+	- create a variable to hold the sum total of all peak loads in the system:
+	`system_total_peak_load = 0`
+	- create a variable to hold the sum total of the system area:
+	`system_total_area = 0`
+	- create a list of the EFLH's for each zone `eflh = []`
+	- look through all zones in the building segment:
+	`for zone_b in building_segment.zones:`
+		- check that zone_b is not the zone being investigated:
+		`if zone_b != zone:`
+			- add the zone internal loads to the system_total_peaks:
+			`a = get_zone_peak_internal_load(zone_b)`
+			`system_total_peak += a[0]`
+			`system_total_area += a[1]`
```

Comment:
For floor area I would use get_hvac_zone_list_w_area function.

Reply:
good idea.  Changed